hw_2.js

```diff
@@ -0,0 +1,116 @@
+"use strict"
+
+// // Task 1
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+
+// console.log(x + y + z);
+
+// // Task 2
+
+// let seconds = 60;
+// let minutes = 60;
+// let hours = 24;
+// let days = 365;
+// let age = 25;
+
+// let myAgeInSeconds = seconds * minutes * hours * days * age;
+
+// console.log(myAgeInSeconds);
+
+// Task 3
+
+// let count = 42;
+// let userName = '42';
+
+// count = String(count); // Метод 1
+// console.log(count);
+// console.log(typeof count);
+
+// count = count.toString(); // Метод 2
+// console.log(count);
+// console.log(typeof count);
+
+// userName = Number(userName); // Метод 1
+// console.log(userName);
+// console.log(typeof userName);
+
+// userName = userName * 1; // Метод 2
+// console.log(userName);
+// console.log(typeof userName);
+
+// Task 4
+
+// let a = 1;
+// let b = 2;
+// let c = "белых медведей";
+
+// let twelveBears = a + b + " " + c;
+// console.log(twelveBears);
```

Comment:
Пропустил(
тут будет 3 белых медведей
можно так
```suggestion
// let twelveBears = `${a}${b} ${c}`;
// console.log(twelveBears);
```

Reply:
Ну вроде всё)